libs/libjaylink/Makefile

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright (C) 2023 OpenWrt.org
```

Comment:
Unless you have a copyright assignment to openwrt on file, you can't do this.  This shoudl either be omitted, or your own name. see the first bullet hgere: https://github.com/openwrt/packages/blob/master/CONTRIBUTING.md#makefile-contents-should-contain